### PR TITLE
Additions to DG.

### PR DESCRIPTION
**Pull request by aditi2313**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -645,10 +709,10 @@ testers are expected to do more *exploratory* testing.
...
    1. Test case: `book delete 1`<br>
       Expected: First book is deleted from the list. Details of the deleted book shown in the status message. Timestamp in the status bar is updated.
-   
+
    1. Test case: `book delete 0`<br>
       Expected: No book is deleted. Error details shown in the status message. Status bar remains the same.
-   
+
    1. Other incorrect delete commands to try: `book delete`, `book delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
```
Comment (Halpfrog) [inline docs/DeveloperGuide.md:718]: Adds 2 new use cases

File: docs/UserGuide.md [hunk 2]
```diff
@@ -304,7 +304,7 @@ AddressBook data are saved in the hard disk automatically after any command that
 
 AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
 
-<div markdown="span" class="alert alert-warning">:exclamation: 
+<div markdown="span" class="alert alert-warning">:exclamation:
 **Caution:**
 If your changes to the data file makes its format invalid, AddressBook will discard all data and start with an empty data file at the next run.
 </div>
```
Comment (Halpfrog) [inline docs/UserGuide.md:310]: Removed trailing whitespace

File: docs/team/yl-ang.md [hunk 3]
```diff
@@ -32,6 +32,5 @@ Given below are my contributions to the project.
 
 * **Community**:
   * PR reviewed (with non-trivial review comments): [\#101](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/101), [\#100](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/100), [\#96](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/96), [\#95](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/95), [\#83](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/83), [\#79](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/79), [\#78](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/78), [\#73](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/73), [\#69](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/69), [\#66](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/65), [\#62](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/65), [\#60](https://github.com/AY2122S2-CS2103T-W14-1/tp/pull/60)
-  *
 
-* **Tools**:
\ No newline at end of file
+* **Tools**:
```
Comment (Halpfrog) [inline docs/team/yl-ang.md:36]: LGTM

File: docs/team/yuqitanyq.md [hunk 4]
```diff
@@ -63,11 +63,11 @@ Given below are my contributions to the project.
...
-  
+
 
 * **Community**:
   * Raised issues on the forum if necessary and gave detailed descriptions of problems.
-  
+
 
 * **Tools**:
   * Gradle
```
Comment (Halpfrog) [inline docs/team/yuqitanyq.md:73]: LGTM, thanks for clearing trailing whitespace

**Review by Halpfrog — COMMENTED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -577,6 +577,70 @@ Extension
...
+    * 3a1. LibTask shows an error message.
+
+      Use case resumes from step 3.
+
+* 3b. The book already has 3 requesters.
+
+    * 3a1. LibTask shows an error message.
+
+      Use case resumes from step 3.
+
```
Comment (yl-ang) [inline docs/DeveloperGuide.md:643]: MSS added is relevant :100:

**Review by yl-ang — APPROVED** [review]
LGTM!

**Pull request merged**